tests/test_result_to_dict.py

```diff
@@ -35,6 +35,14 @@ class Data(Base):
     data_enum = Column(Enum(MyEnum))
 
 
+def convert_str_to_dict(result):
+    obj = result.get('object')
+    if obj and isinstance(obj, str):
+        result['object'] = json.loads(obj)
+        return result
+    return None

```

Comment:
for the code being clear. 
this weill `return obj`

for the last test case, 
`def convert_return_none(result):
    return None
`